samples/portability/portability.rst

```diff
@@ -8,5 +8,3 @@ Portability Samples
    :glob:
 
    **/*
-
```

Comment:
Not sure what this file is there for. You are only deleting empty lines here in this commit.